contracts/src/chainlink/cairo/access/ownable.cairo

```diff
@@ -72,6 +72,8 @@ namespace Ownable:
         end
         assert_only_owner()
         Ownable_proposed_owner.write(new_owner)
+        let (previous_owner : felt) = Ownable_owner.read()
+        OwnershipTransferred.emit(previous_owner, new_owner)
```

Comment:
Nice catch! 

Reading this contract, looks like we have some issues with `transfer_ownership` fn which also emits `OwnershipTransferred` event while it shouldn't! Check out these as a guideline:

- https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/ConfirmedOwnerWithProposal.sol
- https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable2Step.sol

Let's actually add the new type of event `event OwnershipTransferRequested(address indexed from, address indexed to);` and only emit that one in external `transfer_ownership` fn.

Reply:
Also I feel we should rename the internal `_transfer_ownership` as `_accept_ownership` or `_accept_ownership_transfer`.

The current implementation of `_transfer_ownership` doesn't match what an external `transfer_ownership` does (just proposes, 2-step) so it's confusing.